packages/flutter_tools/test/src/common.dart

```diff
@@ -347,3 +348,36 @@ FakeAnalytics getInitializedFakeAnalyticsInstance({
     clientIde: clientIde,
   );
 }
+
+/// This utility function allows us to check for an instance of
```

Comment:
Please ensure all dartdocs start with a single sentence separated from the rest of the text:

https://dart.dev/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary

https://dart.dev/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph